sources/packages/web/src/components/common/StudentApplication.vue

```diff
@@ -1,15 +1,17 @@
 <template>
   <formio
     :formName="selectedForm"
-    :data="formData"
+    :data="{ ...initialData, ...additionalFormData }"
```

Comment:
For my understanding, I did not follow this change. Since the `StudentApplicationFormData` already has the `howWillYouBeAttendingTheProgram`, what was the benefit in assigning it using two objects.